tests/lib/prepare-project.sh

```diff
@@ -32,6 +44,28 @@ build_deb(){
     cp ../*.deb "$GOHOME"
 }
 
+fedora_build_rpm() {
```

Comment:
That isn't that easy. The build process looks different on Fedora and openSUSE. On Fedora we really want to utilize mock to ease as much as possible of the actual package build but on openSUSE we have to stick with the plain old rpm command. We can write a super generic part of this but it might get too generic in the end with all the different corner cases (vendoring vs. no vendoring, mock vs. rpm, package build location, ...). I tend to start with a specialized version of both and see where we end up and then refactor if there is a need to.